resources/lib/KodiHelper.py

```diff
@@ -87,12 +87,31 @@ def show_search_term_dialog (self):
         :obj:`str`
             Term to search for
         """
-        dlg = xbmcgui.Dialog()
-        term = dlg.input(self.get_local_string(string_id=30003), type=xbmcgui.INPUT_ALPHANUM)
-        if len(term) == 0:
-            term = ' '
+        term = xbmcgui.Window(xbmcgui.getCurrentWindowId()).getProperty('searchstring')
```

Comment:
Maybe the existing cache helper methods could be reused? https://github.com/asciidisco/plugin.video.netflix/blob/master/resources/lib/KodiHelper.py#L286-L327

Reply:
I have some issues caused by cachhelper so it should be rechecked if it makes sense in current version (not refreshing lists etc.) and the search string is only a single string so i have made it standalone..